pkg/internal/imports/gen.go

```diff
@@ -16,5 +18,6 @@
 
 package imports
 
-//go:generate bash -c "cd $(mktemp -d) && GO111MODULE=on go get github.com/edwarnicke/imports-gen@v1.0.0"
-//go:generate ${GOPATH}/bin/imports-gen
+//go:generate bash -c "rm -rf imports*.go"
+//go:generate bash -c "cd $(mktemp -d) && GO111MODULE=on go get github.com/edwarnicke/imports-gen@v1.1.0"
```

Comment:
I'd recommend using v1.1.2 of imports-gen

Reply:
https://github.com/edwarnicke/imports-gen/releases/tag/v1.1.2

Reply:
ok, fixed!